types/zeit__next-typescript/index.d.ts

```diff
@@ -0,0 +1,14 @@
+// Type definitions for @zeit/next-typescript 0.1
+// Project: https://github.com/zeit/next-plugins/tree/master/packages/next-typescript
+// Definitions by: icopp <https://github.com/icopp>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+// TypeScript Version: 2.6
+
+import { ServerConfig } from 'next';
+
+declare function withTypescript(
+    /** @default {} */
```

Comment:
If this has a default value, it should be optional?

Reply:
D'oh! You are correct.